prow/cmd/peribolos/main_test.go

```diff
@@ -1077,7 +1077,7 @@ func TestConfigureTeam(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
 			fc := makeFakeTeamClient(tc.github)
-			err := configureTeam(fc, fakeOrg, tc.teamName, tc.config, tc.github)
+			err := configureTeam(fc, fakeOrg, tc.teamName, tc.config, tc.github, 0)
```

Comment:
Will you `s/0/tc.parentID` and ensure cover this in "do not patch team when values are the same" and have a "patch team when parent changes"